packages/common/src/interfaces.ts

```diff
@@ -16,6 +18,29 @@ export type WorkflowQueryType = (...args: any[]) => any;
 export type Workflow = (...args: any[]) => WorkflowReturnType;
 
 declare const argsBrand: unique symbol;
+declare const retBrand: unique symbol;
+
+/**
+ * An interface representing a Workflow update definition, as returned from {@link defineUpdate}
+ *
+ * @remarks `Args` can be used for parameter type inference in handler functions and *WorkflowHandle methods.
```

Comment:
`*WorkflowHandle` why the `*`?

Reply:
Hm, not sure, the asterisk was inherited from the Signal and Query docstrings. Removed from all 6827d7730357585e73ea47041cb63601bdf852ee